pkg/api/v1/server.go

```diff
@@ -92,5 +99,9 @@ func (s *Server) toDBModel() (*models.Server, error) {
 		dbS.ID = s.UUID.String()
 	}
 
+	if s.DeletedAt == nil {
+		dbS.DeletedAt = null.Time{}
+	}
+
```

Comment:
```suggestion
```

We don't allow manually changing the deleted at value via an update only a DELETE call, so this isn't needed.